.github/workflows/discord-notifications.yml

```diff
@@ -0,0 +1,215 @@
+name: Discord PR Notifications
+
+on:
+  pull_request:
+    types:
+      - opened
+      - closed
+      - reopened
+      - ready_for_review
+      - synchronize
```

Comment:
I think this will provide too much noise. Can't we just set utilize the mark PR as ready for review when it is actually ready for review?